veps/sig-network/passt-migration-proposal.md

```diff
@@ -0,0 +1,253 @@
+# VEP #21: Seamless TCP migration and vhost-user support with passt
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, linking to VEP directory in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+Kubevirt users can benefit from seamless live migration of VMs without disconnection of TCP sessions, while using `passt`.
```

Comment:
KubeVirt

Rather than the "without *dis*connection" double negative, I'd go with: "preserving TCP connections"